src/gsy_e/models/area/markets.py

```diff
@@ -98,9 +98,7 @@ def update_area_market_id_lists(self) -> None:
 
     def activate_future_markets(self, area: "Area") -> None:
         """Wrapper for activation methods for all future market types."""
-        if ConstSettings.ForwardMarketSettings.ENABLE_FORWARD_MARKETS:
-            self._activate_forward_markets(area)
```

Comment:
Is this check applied somewhere else that I do not recall ? Otherwise I would recommend to reinstate it since the forward markets should not be activated by default yet

Reply:
I see that the _activate_forward_markets includes a similar check therefore this is indeed redundant here.